account_fiscal_position_rule_sale_stock/README.rst

```diff
@@ -0,0 +1,94 @@
+=======================================
+Account Fiscal Position Rule Sale Stock
+=======================================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--fiscal--rule-lightgray.png?logo=github
+    :target: https://github.com/OCA/account-fiscal-rule/tree/10.0/account_fiscal_position_rule_sale_stock
+    :alt: OCA/account-fiscal-rule
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/account-fiscal-rule-10-0/account-fiscal-rule-10-0-account_fiscal_position_rule_sale_stock
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/93/10.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module extends the sale stock.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+No configuration required.
+
+Changelog
+=========
+
+10.0.1.0.0 (2019-08-16)
+~~~~~~~~~~~~~~~~~~~~~~~
+
+* Created module, it's a extraction of account_fiscal_rule_stock to better modularization.
```

Comment:
s/Module creation./It is an extraction from account_fiscal_rule_stock for a better modularization./